cmd/greenmask/cmd/dump/dump.go

```diff
@@ -46,6 +46,7 @@ var (
 			if err != nil {
 				log.Fatal().Err(err).Msg("fatal")
 			}
+			st = st.SubStorage(Config.Dump.PgDumpOptions.DbName,true)
```

Comment:
Use fmt.Sprintf instead. You don't need to call SubStorage two times. Just assemble the prefix and then call